SimTracker/Common/src/SiG4UniversalFluctuation.cc

```diff
@@ -222,7 +207,7 @@ double SiG4UniversalFluctuation::SampleFluctuations(const double momentum,
   //
   if (suma > sumalim)
   {
-    p1 = 0., p2 = 0 ;
+    double p1, p2;
```

Comment:
Hi David, no, because both p1 and p2 are defined below in any case. The static analyser complains to this line.